pkgs/by-name/zf/zfind/package.nix

```
@@ -0,0 +1,26 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "zfind";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "laktak";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-KAkMoCanflMNiAQLyA7yHdcxjlgigpL/naHrvFPbDd0=";
+  };
+
+  vendorHash = "sha256-mmoJrqWRmJTAR2wkAB52mpYEEj3XD+jHvlVrw51vqys=";
+
+  ldflags = [ "-X" "main.appVersion=${version}" ];
+
+  meta = with lib; {
+    description = "zfind allows you to search for files, including inside tar, zip, 7z and rar archives. It makes finding files easy with a filter syntax that is similar to an SQL-WHERE clause.";
```

```suggestion
    description = "zfind allows you to search for files, including inside tar, zip, 7z and rar archives";
```

or move that into longDescription

---

Just to clarify for myself: Having the package without description - but with longDescription - content is fine?